WebNetSample/Controllers/ProductsController.cs

```diff
@@ -1,15 +1,19 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using WebNetSample.Business.Abstract;
+using WebNetSample.Core.Pagination;
+using WebNetSample.Entity.Concrete;
 
 namespace WebNetSample.WebNetMVC.Controllers;
 
 public class ProductsController : Controller
 {
     private readonly IProductService _productService;
+    private readonly PaginationParameters _paginationParameters;
 
-    public ProductsController(IProductService productService)
+    public ProductsController(IProductService productService, PaginationParameters paginationParameters)
```

Comment:
Pagination params should be passed from view/client to endpoint actually. In our case it is better to take this pagination params from app settings, create object in action and pass id directly to service

Reply:
Please see my last commit and let me know what you think [93e6773](https://github.com/osmanaliyardim/WebNetSample/pull/4/commits/93e67738790c8781f05aac6755f6e3faaf597e1a)